src/sql/workbench/parts/notebook/common/models/notebookInput.ts

```diff
@@ -41,9 +46,17 @@ export class NotebookEditorModel extends EditorModel {
 		this._register(this.notebookService.onNotebookEditorAdd(notebook => {
 			if (notebook.id === this.notebookUri.toString()) {
```

Comment:
>notebook.id === this.notebookUri.toString() [](start = 7, length = 43)

I'd double check that this is working correctly after the URI changes - I had to fix issues with this for .sql files since it was escaping path characters such as : 

(both untitled and saved notebooks)

Reply:
I've been testing both untitled/saved nb's on this branch and haven't noticed any erroneous behavior at all related to uri's.